<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StyleMainWindow</class>
 <widget class="QMainWindow" name="StyleMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>666</width>
    <height>278</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>176</width>
      <height>84</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="title">
     <string>按鈕元件</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>小按鈕1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>小按鈕2</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QPushButton" name="pushButton_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>大按鈕</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>10</y>
      <width>171</width>
      <height>81</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="text">
     <string>可選中按鈕</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>99</y>
      <width>176</width>
      <height>131</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="title">
     <string>狀態燈顯示1</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="3" column="0">
      <widget class="QRadioButton" name="fineRadioBtn">
       <property name="text">
        <string>正常</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="unknownRadioBtn">
       <property name="text">
        <string>不明</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="faultRadioBtn">
       <property name="text">
        <string>故障</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="closedRadioBtn">
       <property name="text">
        <string>關閉</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>設備狀態：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="CRoundPushBtn" name="statPushBtn">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>191</x>
      <y>99</y>
      <width>171</width>
      <height>131</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="title">
     <string>狀態燈顯示2</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="3" column="0">
      <widget class="QRadioButton" name="radioButton_4">
       <property name="text">
        <string>正常</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QRadioButton" name="radioButton_6">
       <property name="text">
        <string>不明</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QRadioButton" name="radioButton_7">
       <property name="text">
        <string>故障</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QRadioButton" name="radioButton_8">
       <property name="text">
        <string>關閉</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="locale">
        <locale language="Chinese" country="Taiwan"/>
       </property>
       <property name="text">
        <string>設備狀態：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>30</y>
      <width>171</width>
      <height>22</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <item>
     <property name="text">
      <string>新建項目1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建項目2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>新建項目3</string>
     </property>
    </item>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>70</y>
      <width>221</width>
      <height>111</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="sss">
     <attribute name="title">
      <string>選項頁A</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="sss">
     <attribute name="title">
      <string>選項頁B</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="sff">
     <attribute name="title">
      <string>選項頁C</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="fdd">
     <attribute name="title">
      <string>選項頁D</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>選項頁E</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="text">
     <string>氣溫：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>200</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="text">
     <string>32.6 度</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>666</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_F">
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="title">
     <string>文件(&amp;F)</string>
    </property>
    <addaction name="action_X"/>
   </widget>
   <widget class="QMenu" name="menu_S">
    <property name="locale">
     <locale language="Chinese" country="Taiwan"/>
    </property>
    <property name="title">
     <string>風格(&amp;S)</string>
    </property>
    <addaction name="action_1"/>
    <addaction name="action_2"/>
    <addaction name="action_3"/>
   </widget>
   <addaction name="menu_F"/>
   <addaction name="menu_S"/>
  </widget>
  <action name="action_X">
   <property name="text">
    <string>退出(&amp;X)</string>
   </property>
  </action>
  <action name="action_1">
   <property name="text">
    <string>風格1</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>風格2</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>風格3</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CRoundPushBtn</class>
   <extends>QPushButton</extends>
   <header>RoundPushBtn.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
